app/lib/omniauth/strategies/moj_oauth2.rb

```diff
@@ -59,8 +59,20 @@ def client
         hash
       end
 
-      def request_phase
+      def request_phase # rubocop:disable Metrics/AbcSize
```

Comment:
Just thinking about this a bit more. I think we initially added this class as a temporary fix to allow us to debug the access denied issue properly. If we start adding our own logic to it are we accepting that it's now a permanent part of app? 

If so, there's a TODO in`/app/lib/omniauth/strategies/true_layer.rb` which refers to reverting back to the standard OAuth2 strategy after we've finished debugging. That should probably be removed.

Reply:
Quite right.  Will remove it.